src/App.js

```diff
@@ -8,33 +8,38 @@ function App() {
   const [APIData, setAPIData] = useState(false);
   useEffect(() => {
     window.RequestNotifications = RequestNotifications;
+    RequestNotifications();
   }, []);
   
   const RequestNotifications = (url) => {
```

Comment:
Can we use `process.env.NOTIFICATION_URL` as default value for url?

Reply:
If you see inside the function, we are providing `process.env.NOTIFICATION_URL` as a default [here](https://github.com/DynamoDS/NotificationCenter/pull/21/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R15)

Reply:
@avidit Are you talking about using the default param here?

Reply:
yes . will this work? `const RequestNotifications = (url=process.env.NOTIFICATION_URL) =? {`

Reply:
Not familiar with this syntax having `=?` at the end, but this will: `const RequestNotifications = (url=process.env.NOTIFICATION_URL)`

Reply:
sorry for the typo. I meant `const RequestNotifications = (url=process.env.NOTIFICATION_URL) => {`

Reply:
Yes that will work